README.md

```diff
@@ -320,6 +320,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Akanni Modupe](https://github.com/goketech)
   - [Akash Raj](https://github.com/akashrajakku)
   - [Akash Raj](https://github.com/Akashraj161)
+  - [Akash Mishra](https://github.com/helloakash1701)
```

Comment:
I believe your name must be two lines above (`M` comes first later on `R` dos).
Let us know if you need any help doing that. 🙂